mim/commands/download.py

```diff
@@ -35,10 +35,17 @@
     cls=OptionEatAll,
     required=True,
     help='Config ids to download, such as resnet18_8xb16_cifar10')
+@click.option(
+    '--no-check-certificate',
```

Comment:
how about using `--verify/--no-verify'. The usage can be referred to https://github.com/open-mmlab/mim/blob/fae573e4aba32a96b95f94ee36561843e7a2314d/mim/commands/search.py#L58

Reply:
Then how about using '--ignore-ssl' instead?

Reply:
> Then how about using '--ignore-ssl' instead?

It sounds good.